common/djangoapps/edxmako/shortcuts.py

```diff
@@ -55,7 +55,7 @@ def marketing_link(name):
             return reverse(link_map[name])
     else:
         log.debug("Cannot find corresponding link for name: %s", name)
-        return '#'
+        return '/'
```

Comment:
@clrux this is causing test failures, because code assumes "#" as `null url` and overrides url (e.g. with microsite configurations) only when url returned by marketing site is `null` i.e. "#".